sycamore/transforms/partition.py

```diff
@@ -49,6 +49,91 @@ def partition(self, document: Document) -> Document:
         pass
 
 
+class UnstructuredPPTXPartitioner(Partitioner):
+    """
+    UnstructuredPPTXPartitioner utilizes open-source Unstructured library to extract structured elements from
+    unstructured PPTX files.
+
+    Args:
+        include_page_breaks: Whether to include page breaks as separate elements.
+        strategy: The partitioning strategy to use ("auto" for automatic detection).
+        infer_table_structure: Whether to infer table structures in the document.
+        ocr_languages: The languages to use for OCR. Default is "eng" (English).
+        max_partition_length: The maximum length of each partition (in characters).
+        include_metadata: Whether to include metadata in the partitioned elements.
+
+    Example:
+         .. code-block:: python
+
+            pptx_partitioner = UnstructuredPPTXPartitioner(
+                include_page_breaks=True,
+                strategy="auto",
+                infer_table_structure=True,
+                ocr_languages="eng",
+                max_partition_length=2000,
+                include_metadata=True,
+            )
+
+            context = sycamore.init()
+            pdf_docset = context.read.binary(paths, binary_format="pptx")
+                .partition(partitioner=pptx_partitioner)
+
+    """
+
+    @staticmethod
+    def to_element(dict: dict[str, Any]) -> Element:
```

Comment:
Does the powerpoint partitioner return a bounding box? Do we have to do anything to convert it to the bbox representation?

Reply:
It does not! We will prob have to implement the partitioner if we wanted that.